src/main/java/Duke.java

```diff
@@ -1,10 +1,241 @@
+import java.util.Scanner;
```

Comment:
As mentioned in another comment, would be good if you can provide Java Documentation for all your non-trivial methods and class so that other developers are able to better understand your codebase.